doozer/doozerlib/backend/konflux_image_builder.py

```diff
@@ -220,6 +222,7 @@ def _new_component(name: str, application: str, component_name: str,
                 "name": name,
                 "annotations": {
                     "build.appstudio.openshift.io/pipeline": '{"name":"docker-build-multi-platform-oci-ta","bundle":"latest"}',
+                    "build.appstudio.openshift.io/status": '{"pac":{"state":"disabled"}}',
```

Comment:
Konflux by default seems to be enabling custom build pipelines by default, causing it to raise PRs (like [this](https://github.com/openshift-priv/oauth-proxy/pull/1)). Was looking at annotations, and this seem to be the difference between one that is enabled and one that is disabled.